### PR TITLE
Update to astropy v4

### PR DESCRIPTION
**Pull request by rlopezcoto**
Following the discussion on slack:
https://ctapipe.slack.com/archives/C0160M3K362/p1596115158284400

```py3
astropy>=3,<5
```
may install astropy v3.2.3, which is not compatible with lstchain >= v0.6, so this should solve the problem

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,7 +42,7 @@ def find_scripts(script_dir, prefix):
     version=version,
     packages=find_packages(),
     install_requires=[
-        "astropy>=3,<5",
+        "astropy>3,<5",
```
Comment (MaxNoe) [inline setup.py:45]: using `~=4.0` would be more idiomatic

**Review by MaxNoe — APPROVED** [review]

File: setup.py (hunk 1 above)
Comment (MaxNoe) [reply setup.py:45]: It shows explicitly which version is wanted instead of just excluding which are not

Comment (rlopezcoto) [reply setup.py:45]: and what when we get 4.1?
wouldn't `astropy>3,<5` install the latest one?

Comment (MaxNoe) [reply setup.py:45]: The tilde notation does the same, it allows any version `4.x` greater than the one given.

So `~=4.0` is equivalent to `>=4.0,<5`

Comment (MaxNoe) [reply setup.py:45]: https://www.python.org/dev/peps/pep-0440/

Comment (rlopezcoto) [reply setup.py]: ok, thanks, I updated it!

**Review by MaxNoe — APPROVED** [review]

**Pull request merged**